tests/kernel/mem_protect/mem_protect/src/kobject.c

```diff
@@ -158,7 +158,7 @@ void test_kobject_revoke_access(void *p1, void *p2, void *p3)
 			0, K_INHERIT_PERMS | K_USER, K_NO_WAIT);
 
 
-	k_sem_take(&sync_sem, MSEC(100));
+	k_sem_take(&sync_sem, K_MSEC_TO_TICKS(100));
```

Comment:
Shouldn't we use K_MSEC(100) here as in k_sem_take() the timeout is suppose to be in milliseconds?

Reply:
In that file change K_MSEC_TO_TICKS to K_MSEC(100), yes?